CHANGELOG.md

```diff
@@ -9,6 +9,35 @@ stages](https://github.com/kubernetes/community/blob/master/contributors/devel/s
 
 ## Unreleased
 
+## 0.3.0 - 2019-01-23
+
+### Added
+
+- Add documentation for OVS pipeline.
+- Add Network Policy information to monitoring CRDs for both the Agent and the Controller.
+- Add support for the IPsec ESP protocol for GRE tunnels only; it can be enabled by applying antrea-ipsec.yml instead of antrea.yml.
+- Add framework to develop CLI commands for Antrea; the antctl binary only supports the "version" command at the moment.
```

Comment:
You think we can merge the agent-info PR or not?

Reply:
I don't think it's needed. I do not find it super useful until it can be run out of the Pod, so maybe it can wait until after the Chinese holiday.

Reply:
I think even local CLI is useful for debugging, but I agree the current agent-info is not very useful as not much information included yet.